Add new sample to create user personalized marketing emails

annainspace opened the pull request:
*Issue #, if available:*

*Description of changes:*


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

james-jory reviewed (changes requested):
- This PR looks very similar to [personalized marketing campaign](https://github.com/aws-samples/amazon-personalize-samples/tree/master/next_steps/generative_ai/personalized_marketing_campaign) sample. It's not clear what the differences are. Should the existing sample be renamed and/or this sample renamed?
- Please update the READMEs in the root of repo as well as in the "generative_ai" parent directory of this sample's directory.

annainspace commented:
> * This PR looks very similar to [personalized marketing campaign](https://github.com/aws-samples/amazon-personalize-samples/tree/master/next_steps/generative_ai/personalized_marketing_campaign) sample. It's not clear what the differences are. Should the existing sample be renamed and/or this sample renamed?
> * Please update the READMEs in the root of repo as well as in the "generative_ai" parent directory of this sample's directory.

I have renamed the directory and sample for clarity and updated the readme.
